Libraries/LibWeb/HTML/WindowEventHandlers.h

```diff
@@ -20,7 +20,9 @@
     E(onoffline, HTML::EventNames::offline)                       \
     E(ononline, HTML::EventNames::online)                         \
     E(onpagehide, HTML::EventNames::pagehide)                     \
+    E(onpagereveal, HTML::EventNames::onpagereveal)               \
     E(onpageshow, HTML::EventNames::pageshow)                     \
+    E(onpageswap, HTML::EventNames::onpageswap)                   \
```

Comment:
Here and elsewhere, why is there an 'on' in the event name?

Reply:
woops. ill fix that in a sec.

Reply:
As an aside, the other MR should be closed so we dont have two duplicates open. In future, ideally that original MR would have had the history fixed up (no merge commit) instead of opening a compeltely new one.